src/table/examples.js

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import React from 'react';
+import {Table} from './index.js';
+import examples from './examples-list.js';
+import type {TablePropsT} from './types.js';
+
+let data = [
+  ['1', 'Sarah', 'Brown', 31, 'New York No. 1 Anywhere'],
+  ['2', 'Jane', 'Smith', 32, 'San Francisco No. 1 Anywhere'],
+  ['3', 'Joe', 'Black', 33, 'Sydney No. 1 Anywhere'],
+  [
+    '4',
+    'Jane',
+    'Red',
+    34,
+    'London No. 1 Anywhere London No. 1 Anywhere London No. 1 Anywhere London No. 1 Anywhere London No. 1 Anywhere London No. 1 Anywhere London No. 1 Anywhere',
+  ],
+  ['4', 'Jane', 'Red', 34, 'London No. 1 Anywhere'],
+  ['4', 'Jane', 'Red', 34, 'London No. 1 Anywhere'],
+  ['4', 'Jane', 'Red', 34, 'London No. 1 Anywhere'],
+  ['4', 'Jane', 'Red', 34, 'London No. 1 Anywhere'],
+  ['4', 'Jane', 'Red', 34, 'London No. 1 Anywhere'],
+  ['4', 'Jane', 'Red', 34, 'London No. 1 Anywhere'],
+].map(row => [`${row[1]} ${row[2]}`, row[3], row[4]]); // selects data to display
+
+for (let i = 0; i < 5; i++) {
+  data = data.concat(data);
+}
+const ControlledTable = (props: TablePropsT) => {
+  const {overrides, ...otherProps} = props;
+  return <Table {...otherProps} overrides={overrides} />;
+};
+
+export default {
```

Comment:
can you add more examples, potentially one with sorting too?

Reply:
sorting will be added in a follow-up pr